### ISSUE TITLE
Cable Stripping

### ISSUE THREAD

**Issue by Tyler799** [GTNewHorizons/GT-New-Horizons-Modpack#9535]
### Your GTNH Discord Username

Night Thastus#6641

### Your Pack Version

2.1.2.1

### Your Proposal

Add some method other than the HV macerator of recovering both the rubber _and_ metal from cables.

Currently, none of the following work for retrieving the rubber:

* Wire Cutters (for placed blocks, or in crafting)
* Unpackager
* Regular or Fluid extractor
* Disassembler
* Arc Furnace 

I think that having a method to strip cables to wire and give back the rubber would be useful. 

I think one of these 5 (or perhaps a new cheap hand tool like "the wire stripper") could do the job.

### Your Goal

As people move up in tiers and amperage, they may find they have accumulated cables which no longer are useful. While getting back the metal is fairly easy (Arc furnace, fluid extractor, unpackager, alloy smelter for smaller wires), getting back the rubber is not. 

Considering re-use and environmental factors is already a part of the pack in other areas, I think this seems like a reasonable extension of that. Why burn rubber into nasty chemical gas when you could just cut it off with a knife?

### Your Vision

See above.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this feature please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I believe there is nothing similar in the pack already, or the existing solution isn't good enough.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members thinking it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by YannickMG** [comment]
If this is seen as desired, and based on [this real world example](https://www.youtube.com/watch?v=l2H5gHITGGg), this might be a good fit for a Cutting Machine recipe.

Otherwise maybe giving the Recycler a second output slot and a use in recycling these.

**Comment by Glease** [comment]
Unpackager exists.

**Comment by Tyler799** [comment]
> Unpackager exists.

You may not have read the post. Unpackager does not give back the rubber, which was the point of the post.

**Comment by YannickMG** [comment]
Also unpackager doing anything remotely useful is kind of out of character for that machine.

**Comment by vovaksenov** [comment]
Initially, it did give the rubber back. But when silicone and s-b rubber were added, it became too complicated to bother with. Which kind of rubber do you want to receive back? If we have a default, it may create loopholes (create silicone out of normal) or be useless (the other way around). If 2 extra types of cables are added, it would just make it unnecessarily confusing.

**Comment by YannickMG** [comment]
Non-trivial but this could perhaps tie in with the current pollution discussion on introducing waste handling mechanics.

Perhaps instead of giving back same rubber that was used, all types of cable would give back a special kind of rubber waste that can be used in any recipe that takes "recycled rubber".

Other methods of cable stripping that don't give back this rubber waste are assumed to disperse it into the environment as additional pollution.

**Comment by vovaksenov** [comment]
Well. Since recycled rubber will be able to be made from normal rubber, considering the existence of stickreed, its value will be questionable.

**Comment by vovaksenov** [comment]
Maybe if recycled rubber could only be made from IV+ cables (require silicone/s-b), there would be some reason. The problem then becomes the amount you should get back, as the material costs are not as simple for insulating these cables, especially taking into account possibilities to use polyvinyl chloride etc.

**Comment by Tyler799** [comment]
> Initially, it did give the rubber back. But when silicone and s-b rubber were added, it became too complicated to bother with. Which kind of rubber do you want to receive back? If we have a default, it may create loopholes (create silicone out of normal) or be useless (the other way around). If 2 extra types of cables are added (to make it 1 per each possible insulation), it would just make it unnecessarily confusing. Just use superconductors.

If this is the case, why does the macerator still give back rubber pulp from cables? 

I can see why alternative rubbers would make it problematic though. I had no idea, as I am only in MV. (About to hit HV)

**Comment by vovaksenov** [comment]
> If this is the case, why does the macerator still give back rubber pulp from cables?

Most likely unintended. Recipes were not changed when the packager recipes were. Normal rubber is one of the easiest resources to automate and the difficulty of silicone rubber pales in comparison to polybenzimidazole, which has to be automated not too far further from it into the progression. For now, just try to breed stickreed from normal sugar cane. It might seem tedious, but with a hand-held scanner it is not too bad and will save you a lot of time in the long run. 
If you want, you can also do some research into crop robots, which will do most of the work for you, and iirc you only need HV for that.
What I mean by this is, recycling rubber is probably not worth it.

**Comment by bombcar** [comment]
You can also just plant slime trees and get some sulfur.

**Comment by Tyler799** [comment]
I have plenty of sulfur and rubber dust, it was just more a matter of annoyance. I had went through the trouble of making so many wires, now most were useless. I wanted to get back what I had spent and clear out that chest full of wires. Luckily, HV macerator isn't patched yet and I just hit HV, so I guess I'll do that.

**Comment by NeveHanter** [comment]
I still see an use in this, I've recycled nearly all my pre-EV cables and as I'm entering IV and making my power distribution I noticed that I wouldn't be able to recycle proper materials in the future. I get that at the point when I would want to upgrade my main power distribution network I would probably no longer care about the small amounts of materials lost for these wires but this is clearly non-consistent through the tiers.

As an example let's take a platinum cables (IV) that require silicone/styrene-butadiene rubber and additional platinum/polyphenylene foil to make with the most basic recipe but when macerated yield normal rubber that cannot be reused to make larger or different cables from that tier, also the foil material is lost, not even counting the additional PVC/Polydimethylsiloxane pulps that might have been used. Similarly fluid extraction and arc furnace recycling doesn't yield foil materials back but at least they are "consistent" and return ashes (except for the metal foils case and that fluid extractor returns more ash than arc furnace one).

I would suggest either removing the plain rubber recycling from the cables altogether for all the cables so they only recycle back the main wire material or add proper recycling for all the tiers of cables.
